app/response_processor.py

```diff
@@ -71,7 +71,7 @@ def _send_receipt(self, decrypted, xml):
         res_logger = self.logger.bind(request_url=endpoint)
 
         try:
-            res_logger.info("Calling service", service="SDX-MOCK-RECEIPT")
+            res_logger.info("Calling external receipting service", service="External receipt service")
```

Comment:
service tag probably doesn't need service in its value. just service='external receipt' should be fine.